features/event/booking-info.feature

```diff
@@ -253,4 +253,17 @@ Feature: Test the UDB3 events API
         }
       ]
     }
-    """
\ No newline at end of file
+    """
+
+  Scenario: Delete event booking info (with put)
+    Given I create an event from "events/event-with-bookinginfo.json" and save the "url" as "eventUrl"
```

Comment:
There is already an event `event-with-all-fields.json`, maybe reuse that one?

Reply:
As discussed through Slack this test existing file also requires images, so we keep the new smaller event with booking info.